src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2765,6 +2772,16 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-lookup"
+    title={<InlineCode>lookup()</InlineCode>}
+  >
+  You can use the `lookup()` function with a lookup table name to access that table's data in a query. For a tutorial on this, see [How to query lookups](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/lookups).
```

Comment:
Should the link read "Use lookup table data in queries" instead of "How to query lookups", since that's the actual title of the tutorial?